dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/Api/DomainController.php

```diff
@@ -39,26 +39,50 @@ class DomainController extends ApiMutableModelControllerBase
     protected static $internalModelName = 'domain';
     protected static $internalModelClass = '\OPNsense\Bind\Domain';
 
+    # These are here for backwards compatability, should be removed after a bit.
     public function searchMasterDomainAction()
```

Comment:
Can’t we use the implementation of the new function so code doesn’t duplicate?

Reply:
I've done this.